main.js

```diff
@@ -232,6 +232,16 @@ var obstacleLaser = function(x, y) {
   ctx.fillRect(x + 16, 0, 44, 18);
 };
 
+var obstacleRect = function(x, y) {
+  ctx.rect(500, 500, 100, 25);
```

Comment:
Make sure you also use the `x` coordinate. The obstacle is bolted to the floor and that's moving to the left. The `x` coordinate tells the obstacle where the floor is.

Reply:
I have fixed this (disregard this commit)... see new commit.